libs/blocks/hero-marquee/hero-marquee.css

```diff
@@ -480,4 +515,44 @@ html[dir="rtl"] .hero-marquee li.icon-item span.icon {
   /* min height */
   .hero-marquee.s-min-height-desktop { min-height: var(--s-min-height);}
   .hero-marquee.l-min-height-desktop { min-height: var(--l-min-height);}
+
+}
+
+/* Mweb specific */
+@media screen and (max-width: 1199px) {
```

Comment:
What speaks for this strategy as opposed to going with the regular pattern of mobile-first by using `min-width` and overriding for larger viewports? Having min and max width media queries can be confusing and lead to unexpected results that are difficult to debug and fix in the long run. Switching the platform to a mobile-first mentality means to me that we should stick to that pattern of defining the mobile styles and then override what needs overriding for larger viewports.